### PR TITLE
Add DSE support into the test harness

### PR DESCRIPTION
**Pull request by kishkaru**
Adding DSE support into the integration test suite.

### CODE REVIEW

**Comment by stamhankar999** [comment]
I noticed the Travis tests failed, and it's not an rbx thing...though it does look like a setup thing. Ruby updates?

File: cucumber.yml [hunk 1]
```diff
@@ -1,5 +1,14 @@
 <%
 cassandra_version = ENV['CASSANDRA_VERSION'] || '3.0.1'
```
Comment (stamhankar999) [inline cucumber.yml]: Why not 3.1.1 instead of 3.0.1?

**Comment by kishkaru** [comment]
This PR is older, so it doesn't have the latest Travis fixes that you made.

File: cucumber.yml (hunk 1 above)
Comment (kishkaru) [inline cucumber.yml]: This is just a default, as we always specify a C* runtime, this will almost never get used. We're on 3.3.0 now btw.

**Comment by stamhankar999** [comment]
Code looks good; merge at will.

**Pull request merged**